client/components/domains/featured-domain-suggestions/style.scss

```diff
@@ -234,6 +234,8 @@ body.is-section-signup.is-white-signup .step-wrapper .featured-domain-suggestion
 		font-size: $font-title-medium;
 		padding-right: 0;
 		margin-bottom: 0;
+		flex: 3;
+    align-self: baseline;
```

Comment:
Minor: these spaces should be replaced by tabs - dunno why the linter didn't catch this 🤔